src/resources/postcss/base/forms/_tabs.pcss

```diff
@@ -31,87 +65,87 @@
 		@media (--viewport-medium) {
 			display: flex;
 		}
+	}
 
-		button {
-			@mixin form-control-select-button-mobile;
+	.tribe-common-form-control-tabs__button {
+		@mixin form-control-select-button-mobile;
 
-			@media (--viewport-medium) {
-				@mixin hidden;
-			}
+		@media (--viewport-medium) {
+			@mixin hidden;
+		}
 
-			&:after {
-				@mixin form-control-select-button-icon-mobile;
-			}
+		&:after {
+			@mixin form-control-select-button-icon-mobile;
 		}
+	}
 
-		ul {
-			/**
-			 * @todo: remove once listbox JS is implemented
-			 */
-			/* @mixin hidden; */
+	.tribe-common-form-control-tabs__list {
+		/**
+			* @todo: remove once listbox JS is implemented
+			*/
+		/* @mixin hidden; */
 
-			@mixin form-control-select-list-mobile;
+		@mixin form-control-select-list-mobile;
 
-			@media (--viewport-medium) {
-				border: 1px solid var(--color-border-default);
-				box-shadow: none;
-				display: flex !important;
-				flex: none;
-				position: static;
-				visibility: visible;
-				width: auto;
-			}
+		@media (--viewport-medium) {
+			border: 1px solid var(--color-border-default);
+			box-shadow: none;
+			display: flex !important;
+			flex: none;
+			position: static;
+			visibility: visible;
+			width: auto;
 		}
+	}
 
-		li {
+	.tribe-common-form-control-tabs__list-item {
 
-			&:first-child label {
-				@mixin form-control-select-list-item-first;
+		&:first-child label {
+			@mixin form-control-select-list-item-first;
```

Comment:
Noticing in general there are a lot of mixins for what potentially feels like one-off, non-reused styles around specific selectors. What and how are you deciding what to optimize for in how you're going about your use of mixins?

Reply:
yeah, not sure what i was thinking with this (the mixin is just one style), will remove.

Reply:
Fwiw, this question isn't specific to just this instance of a mixin, curious about quite a few of the ones I see.

Reply:
Yup, I'll have another pass at them.

Reply:
So had a quick pass, there are a couple mixins that are only used once (a couple of the `-desktop` variations), which I'm sure could just be put into the class itself. But I figured when I did this that keeping all the styles in one place would be good, even if it's used only once.